docs/source/user-guide/deployment/docker-compose.md

```diff
@@ -27,7 +27,7 @@ There are no officially published Docker images, so it is currently necessary to
 Run the following commands to clone the source repository and build the Docker image.
 
 ```bash
-git clone git@github.com:apache/arrow-ballista.git -b 8.0.0
+git clone git@github.com:apache/arrow-ballista.git -b 0.9.0
```

Comment:
```suggestion
git clone git@github.com:apache/arrow-ballista.git -b 9.0.0
```

Reply:
Did you try running this? it doesn't work.

`fatal: Remote branch 9.0.0 not found in upstream origin`